caesars-palace/blocks/tabs/tabs.js

```diff
@@ -0,0 +1,247 @@
+const MOVE_TABS_FORWARD = 1;
+const MOVE_TABS_BACK = -1;
+
+const TAB_SLIDE_IN_ANIMATION = 'active-tab-slide-in';
+const TAB_SLIDE_OUT_ANIMATION = 'active-tab-slide-out';
+
+const ACTIVE_TAB_TITLE_CLASS = 'active-tab-title';
+const HIDDEN_TAB_TITLE_CLASS = 'hidden-tab-title';
+
+const ACTIVE_TAB_CLASS = 'active-tab';
+const HIDDEN_TAB_CLASS = 'hidden-tab';
```

Comment:
A pattern I like to use is grouping all my selectors or class names into a single object. If you prefer seaprate consts that is OK as well. Sometimes having the object just makes it easier to lookup values in yur IDE.

eg.
```
const classes = Object.freeze({
  tabSlideIn: 'active-tab-slide-in',
  tabSlideOut: 'active-tab-slide-out',
});
```